examples/next/.testRun.ts

```diff
@@ -0,0 +1,36 @@
+import { page, run, urlBase, autoRetry, isGithubAction } from '../../libframe/test/setup'
+
+export { testRun }
+
+function testRun(cmd: 'npm run prod' | 'npm run dev') {
+  if (cmd === 'npm run prod' && isGithubAction()) {
+    test("SKIPED: Next.js prod build doesn't work in GitHub Actions", () => {})
+    return
+  }
+
+  run(cmd, {
+    serverIsRunningMessage: 'started server on',
+    /* Attempt to figure out why Next.js doesn't properly run in GitHub Actions:
+    debug: true,
```

Comment:
@Aslemammad FYI I implemented this `debug` flag, which greatly helps figuring out what is going on in GitHub Actions.